code/modules/food_and_drinks/recipes/food_mixtures.dm

```diff
@@ -50,6 +50,7 @@
 	id = /datum/reagent/consumable/hot_coco
 	results = list(/datum/reagent/consumable/hot_coco = 5)
 	required_reagents = list(/datum/reagent/consumable/milk = 5, /datum/reagent/consumable/coco = 1)
+	required_temp = 320
```

Comment:
Does this mean it needs to be heated in a chem heater or does the microwave work?

Reply:
Chem heater, or you could use the method that everyone in service uses for heating liquids.

Reply:
why would you make a cook's recipe require a chem heater or welder i'm assuming

Reply:
Microwaves also increase the tenperature of reagents